{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2018-14660",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "glusterfs",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "affected versions through 4.1.4"
                                        },
                                        {
                                            "version_value": "affected versions through 3.1.2"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "[UNKNOWN]"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "A flaw was found in glusterfs server through versions 4.1.4 and 3.1.2 which allowed repeated usage of GF_META_LOCK_KEY xattr. A remote, authenticated attacker could use this flaw to create multiple locks for single inode by using setxattr repetitively resulting in memory exhaustion of glusterfs server node."
            }
        ]
    },
    "impact": {
        "cvss": [
            [
                {
                    "vectorString": "6.5/CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
                    "version": "3.0"
                }
            ]
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-400"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "name": "RHSA-2018:3431",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:3431"
            },
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14660",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14660"
            },
            {
                "name": "RHSA-2018:3432",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:3432"
            },
            {
                "name": "RHSA-2018:3470",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:3470"
            },
            {
                "refsource": "GENTOO",
                "name": "GLSA-201904-06",
                "url": "https://security.gentoo.org/glsa/201904-06"
            }
        ]
    }
}